multichain-testing/test/account-balance-queries.test.ts

```diff
@@ -67,7 +67,8 @@ const queryAccountBalances = test.macro({
     });
 
     const offerResult = await retryUntilCondition(
-      () => vstorageClient.queryData(`published.wallet.${agoricAddr}`),
+      () => smartWalletKit.readPublished(`wallet.${agoricAddr}`),
```

Comment:
in what way is this a kit? it looks like 1 object with methods, not a kit of facets.

Reply:
i'm pretty sure "kit" is not necessarily an ExoClassKit.

is there another name you'd suggest? It would be a change to client-utils